build.gradle

```diff
@@ -22,3 +40,24 @@ tasks.register('ktFormat', FormatTask) {
   source files('demo/src', 'library/src')
   report = file('build/reports/ktlint/format-report.txt')
 }
+
+version "1.0.0-alpha.00-SNAPSHOT" // we add it here so that nexus automatically chooses staging or snapshot repo
```

Comment:
We have the version as a hardcoded string in 4 places. Is there a way we can define the version in one place and use a variable everywhere else?

We also have 1.0.1-SNAPSHOT above in `buildscript`. Should that be the same version as here?